pkg/kwokctl/cmd/create/cluster/cluster.go

```diff
@@ -119,6 +119,7 @@ func NewCommand(ctx context.Context) *cobra.Command {
 	cmd.Flags().DurationVar(&flags.Timeout, "timeout", 0, "Timeout for waiting for the cluster to be created")
 	cmd.Flags().DurationVar(&flags.Wait, "wait", 0, "Wait for the cluster to be ready")
 	cmd.Flags().StringVar(&flags.Kubeconfig, "kubeconfig", flags.Kubeconfig, "The path to the kubeconfig file will be added to the newly created cluster and set to current-context")
+	cmd.Flags().StringVar(&flags.Options.CertAddress, "cert-address", flags.Options.CertAddress, "The address to generate pki certificate")
```

Comment:
I'd like to reduce the flags parameter, maybe that's enough ?

`kwokctl create cluster --config ./config.yaml`

``` yaml
# ./config.yaml
kind: KwokctlConfiguration
apiVersion: config.kwok.x-k8s.io/v1alpha1
options:
  kubeApiserverCertSANs:
    - 127.0.0.1
    - localhost
    - 10.0.0.1
```

Reply:
ok, i got it! I will make modifications according to your suggestion.
But i think the flag is needed for test engineer even if used infrequently